pkg/cmd/api/api_test.go

```diff
@@ -702,13 +700,12 @@ func Test_apiRun_inputFile(t *testing.T) {
 			if tt.inputFile == "-" {
 				_, _ = stdin.Write(tt.inputContents)
 			} else {
-				f, err := ioutil.TempFile("", tt.inputFile)
+				f, err := ioutil.TempFile(t.TempDir(), tt.inputFile)
```

Comment:
This calls `t.TempDir()` in a loop. Could we call it just one time outside of the loop and keep reusing the directory until the whole test is done?